docs/excel/performance.md

```diff
@@ -157,6 +157,34 @@ Excel.run(async (ctx) => {
 > [!NOTE]
 > You can conveniently convert a Table object to a Range object by using the [Table.convertToRange()](https://docs.microsoft.com/javascript/api/excel/excel.table#converttorange--) method.
 
+## Untrack unneeded ranges
+
+The JavaScript layer creates proxy objects for your add-in to interact with the Excel workbook and underlying ranges. These objects persist in memory until `context.sync()` is called. Large batch operations may generate a lot of proxy objects that are only needed once by the add-in and can be released from memory before the batch executes.
+
+The [Range.untrack()](https://docs.microsoft.com/javascript/api/excel/excel.range#untrack--) method releases Excel Range objects from memory. Calling this after your add-in is done with the range yields a noticeable performance benefit when using large numbers of Range objects. `Range.untrack()` is a shortcut for [ClientRequestContext.trackedObjects.remove(thisRange)](https://docs.microsoft.com/javascript/api/office/officeextension.trackedobjects#remove-object-). Any proxy object can be untracked by removing it from the tracked objects list in the context. Typically, Range objects are the only Excel objects used in sufficient quantity to justify untracking.
```

Comment:
- Assuming that the `untrack()` method only releases from memory the range for which it is called, you should change "...releases Excel Range objects from memory." -> "...releases an Excel Range object from memory."

- "Calling this after..." -> "Calling this method after..."

- Suggest that you consider either:
    - removing the two sentences "Range.untrack() is a shortcut for ClientRequestContext.trackedObjects.remove(thisRange). Any proxy object can be untracked by removing it from the tracked objects list in the context." altogether -- provided that `Range.untrack()` is functionally-equivalent with `ClientRequestContext.trackedObjects.remove(thisRange)`
        OR 
    - moving these two sentences into a `> [!NOTE]` following the paragraph (that way, the info is still available as an FYI for those who may care), but the section is mainly focusing on `Range.untrack()`